src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -51,20 +51,25 @@ public Exercise(int age) {
         Create a constructor that accepts both a String and an int as parameters, in that order, and assign the values
         provided to the name and age members
      */
-
+    public Exercise(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
 
 
     /*
         2. Create a method named add that accepts two integers. The method should return the numbers added together.
      */
-
+    public int add(int one, int two) {
+        return one + two;
+    }
 
 
     /*
         3. Create another method named add that accepts two Strings. The method should return the strings concatenated
         together with a space in between.
      */
-
-
-
+    public String add(String Hello, String World) {
```

Comment:
Hi Esther

Everything else works and this part (adding together two strings) will pass the test as it stands because it returns the correct string:

```java
    public String add(String Hello, String World) {
        return "Hello" + " " + "world";
    }
```
but it should return the values contained in the two arguments to the method joined together with a string instead. So removing the quote marks around `Hello` and `world` would fix this (although I think you'll need to give `world` a capital letter so that it matches the argument name. If we changed the values that the method was called with in the test then this would fail.

Cheers
Dave